servers/visual/visual_server_canvas.cpp

```diff
@@ -51,17 +51,34 @@ void VisualServerCanvas::_render_canvas_item_tree(Item *p_canvas_item, const Tra
 	}
 }
 
+void _collect_ysort_children(VisualServerCanvas::Item *p_canvas_item, Transform2D p_transform, VisualServerCanvas::Item **r_items, Transform2D *r_extra_transforms, int &r_index) {
```

Comment:
Just a style nitpick: the rest of the class seems to use `Item` directly, do we need the explicit namespace here?

Reply:
Yes, since the function is not part of the class. It won't compile without a namespace there.